website/versioned_docs/version-v11.0.0/guided-tour/rendering/fragments.md

```diff
@@ -85,7 +85,7 @@ Let's distill what's going on here:
 * A *fragment reference* is an object that Relay uses to *read* the data declared in the fragment definition; as you can see, the `UserComponent_user` fragment itself just declares fields on the `User` type, but we need to know *which* specific user to read those fields from; this is what the fragment reference corresponds to. In other words, a fragment reference ** is like *a pointer to a specific instance of a type* that we want to read data from.
```

Comment:
Do you mind removing these stray `**`s too?

Reply:
Thanks for pointing that out. All `*`s and `**`s are dropped! https://github.com/facebook/relay/pull/3408/commits/8b21be8616af5ebccaddcc50195bcf88a144ff0c